ding/reward_model/trex_reward_model.py

```diff
@@ -44,6 +51,24 @@ def cum_return(self, traj):
         x = x.reshape(-1, 784)
         x = F.leaky_relu(self.fc1(x))
         r = self.fc2(x)
+        return r
+
+
+class TREX_Model(nn.Module):
+
+    def __init__(self, obs_shape):
+        super(TREX_Model, self).__init__()
+        if isinstance(obs_shape, int) or len(obs_shape) == 1:
+            self.encoder = FCEncoder(obs_shape, [512, 64])
+        # Conv Encoder
+        elif len(obs_shape) == 3:
+            self.encoder = ConvEncoder()
+
+    def cum_return(self, traj):
+        '''calculate cumulative return of trajectory'''
+        sum_rewards = 0
+        sum_abs_rewards = 0
```

Comment:
It doesn't seem necessary to use these two variables?

Reply:
abs_reward will be used in the future exploration (which we may not have time to do)